catalogue/webapp/utils/works.ts

```diff
@@ -71,19 +71,6 @@ export function getDownloadOptionsFromImageUrl(
   }
 }
 
-export function getEncoreLink(work: Work): string | undefined {
-  const sierraWorkIds = getWorkIdentifiersWith(work, {
```

Comment:
It looks like this is the only use of `getWorkIdentifiersWith` (but linting doesn't catch it as unused because it's exported). Let's delete that function too.